bin/runtime-common/src/messages.rs

```diff
@@ -1401,4 +1406,38 @@ mod tests {
 			Err(target::MessageProofError::MessagesCountMismatch),
 		);
 	}
+
+	#[test]
+	fn transaction_payment_works_with_zero_multiplier() {
+		assert_eq!(
+			transaction_payment(
+				100,
+				10,
+				FixedU128::zero(),
+				|weight| weight,
+				MessageLaneTransaction {
+					size: 50,
+					dispatch_weight: 777
+				},
+			),
+			100 + 50 * 10,
```

Comment:
nit: Having something like `base_weight + size * per_byte_fee` here would be nice

Reply:
Then it'll be duplicate of the `transaction_payment` itself :) What it does is simply gives names to these numbers